### ISSUE TITLE
Add list view to images 

### ISSUE THREAD

**Issue by fzyang1227** [Code-4-Community/speak-for-the-trees-frontend#293]
**Blocked by**
#292 

**Why**
- ability to view uploaded images to display on the approval page
- see #316 

**What to do**
- connect data for api route _(make sure to note the name for the route for #316 )_
- implement a list view of the unapproved images to be reviewed
- implement the component for this list section
- refer to [figma design](https://www.figma.com/file/kDLySmyjpw2dorjTkuTuBB/SFTT-2023-2024?type=design&node-id=1196-4068&mode=design&t=kbLCz3L7kGNSDVJu-4)
    - _notes: can ignore the recently rejected section_
    - filters include site_id and date submitted (no need for the status anymore, also for the list view no need for status)
    - the following two views by getting all the relevant info for the images that are unapproved (see: https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/issues/202)

*specific features to note:*
1. the top with column names to sort by that feature
2. filters to filter the unapproved images
3. _(in list view)_ click on any space (other than the mini preview) to add to be approved

**popups for expanding on the image will be implemented in another ticket**

![Image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/62119227/8638315d-2689-49d8-8e98-b41ec8487bec)
